### PR TITLE
CNV-13660: RN Transfer NIC static IP to bridge

### PR DESCRIPTION
**Pull request by sjhala-ccs**
[CNV-13660](https://issues.redhat.com/browse/CNV-13660)

Technology Preview feature

Preview build: https://deploy-preview-42507--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-10-release-notes.html#virt-4-10-technology-preview

### CODE REVIEW

File: virt/virt-4-10-release-notes.adoc [hunk 1]
```diff
@@ -69,12 +69,10 @@ The SVVP Certification applies to:
 
 //CNV-11201 Rollout of NodeNetworkConfigurationPolicy
 
-//CNV-13660 Inherit static IP from a NIC attached to the bridge
 
 //CNV-13679 SR-IOV live-migration without privilege escalation
 * xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[Live migration] is now supported by default for virtual machines that are attached to an SR-IOV network interface.
 
-//CNV-13680 Primary and secondary networks over a shared NIC on OVN clusters
```
Comment (AlonaKaplan) [inline virt/virt-4-10-release-notes.adoc:77]: Seems that this removal is not related to the PR.

Comment (sjhala-ccs) [reply virt/virt-4-10-release-notes.adoc:77]: Correct. This feature has been pushed to 4.11, so I just deleted the placeholder for the release note.

**Comment by ousleyp** [comment]
LGTM
